src/ParallelAlgorithms/DiscontinuousGalerkin/InitializeMortars.hpp

```diff
@@ -199,8 +199,8 @@ struct InitializeMortars {
 
   template <typename DbTagsList>
   static db::DataBox<DbTagsList> initialize_next_temporal_id_on_mortars(
-      db::DataBox<DbTagsList>&& box, std::false_type /* meta */) noexcept {
-    return std::forward<db::DataBox<DbTagsList>>(box);
+      db::DataBox<DbTagsList>&& box, std::false_type /* disabled */) noexcept {
```

Comment:
the comment should be the same in both sites (i.e. it is true above and false here)